src/main/scala/sbtlicensereport/SbtLicenseReport.scala

```diff
@@ -106,25 +110,26 @@ object SbtLicenseReport extends AutoPlugin {
         )
         Seq(config)
       },
+      dumpLicensePruneDuplicates := true,
```

Comment:
Default setting should probably go into `globalSettings` to have the most flexibility - https://www.scala-sbt.org/1.x/docs/Plugins-Best-Practices.html#Provide+default+values+in

Reply:
I can't imagine a situation where people would want to set this setting to `false` though. Does this need to be a setting?

Reply:
> Default setting should probably go into globalSettings to have the most flexibility

Done

> I can't imagine a situation where people would want to set this setting to false though. Does this need to be a setting?

Only thing I can come up with is debugging, but its not a great answer. I can also remove it if you want.

Reply:
Yea. Let's remove it.

Reply:
Done, also renamed commit to be more precise in its wording.